### PR TITLE
Refactor requirements installation and dependabot config

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,13 +6,21 @@ updates:
...
   schedule:
     interval: daily
     time: "03:00"
+  reviewers:
+    - neuro-inc/mlops
   open-pull-requests-limit: 10
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
```
Comment (anayden) [inline .github/dependabot.yml:22]: Do we need this daily? Maybe weekly?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -6,13 +6,21 @@ updates:
     interval: daily
     time: "03:00"
   open-pull-requests-limit: 10
-  assignees:
-  - anayden
-  - artemlops
-  - YevheniiSemendiak
+  reviewers:
+    - neuro-inc/mlops
```
Comment (anayden) [inline .github/dependabot.yml:10]: Great idea to use the team reference here 👍

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -33,7 +41,7 @@ jobs:
         run: |
           python -m venv venv
           source venv/bin/activate
-          pip install -U neuro-cli packaging neuro-sdk
+          pip install -U neuro-cli neuro-sdk
```
Comment (anayden) [inline .github/workflows/ci.yaml:44]: by the way, I imagine `neuro-cli` installs `neuro-sdk` as well.

**Review by anayden — APPROVED** [review]
LGTM. A few very minor things

File: .github/dependabot.yml (hunk 1 above)
Comment (YevheniiSemendiak) [reply .github/dependabot.yml:22]: Personally, I don't see any issues with this, tbh.
We don't pay for it, neither run jobs, afaik.

On the other hand, imagine `neuro-cli` release, and on the next day, we get just a bunch of PRs, all of which will be merged within the same context (if there was a problematic CLI release - we all will keep in mind that this should not be merged yet).

Comment (anayden) [reply .github/dependabot.yml:22]: Ok, that makes sense

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (YevheniiSemendiak) [reply .github/workflows/ci.yaml]: rmved in 24d9ab4

**Pull request merged**